obspy/signal/detrend.py

```diff
@@ -23,11 +23,16 @@ def simple(data):
     point of the trace
 
     :param data: Data to detrend, type numpy.ndarray.
-    :return: Detrended data.
+    :return: Detrended data. Returns the original array which has been
+        modified in-place.
```

Comment:
`np.require` will make a copy if changing the `dtype`, so this is not always true.